### DISCUSSION TITLE
Training with CPU only?

### DISCUSSION THREAD

**Discussion by kuldp18** [bmaltais/kohya_ss#679] [General]
I recently saw this video : https://www.youtube.com/watch?v=TpuDOsuKIBo for LoRA training and I was wondering if I can really train a Lora with CPU only since I have only 4gb vram in my graphics card.

I know there is a "cpu only" prompt while installing but I want to know if the results would be as good as using a GPU. I have trained few Lora models using colab but "CPU only" is the only way for me locally. So kindly share your thoughts.

**Comment by DCVirtualCosmos** [comment]
i have tried stable diffusion with the automatic1111 webui on cpu only for the lols and must say I don't recommend it. To produce the same image my 2070 super can do in 15 seconds, on CPU only took 10 minutes. I thought well but al least I can leave the pc building a huge resolution image, and my pc ended crashing lmao

**Reply by kuldp18** [reply]
So I think it is not a good idea to train on CPU haha, I have to rely on Google colab then.
